dspace-api/src/main/java/org/dspace/app/itemimport/ItemImportServiceImpl.java

```diff
@@ -466,6 +468,7 @@ protected Item addItem(Context c, List<Collection> mycollections, String path,
         String mapOutputString = null;
 
         System.out.println("Adding item from directory " + itemname);
+        log.debug("adding item from directory " + itemname);
```

Comment:
Understood now. Yes, the duplicate logging seems to be needed for that scenario. At some point, it might be best to rip out all the `System.out.println` calls (replacing them with logging), and just report to check the logs